packages/core/helper-plugin/lib/src/hooks/useTracking/index.js

```diff
@@ -11,7 +11,7 @@ const useTracking = () => {
       try {
         axios.post('https://analytics.strapi.io/track', {
           event,
-          properties: { ...properties, projectType: strapi.projectType },
+          properties: { ...properties, projectType: strapi.projectType, environment: process.env.NODE_ENV },
```

Comment:
Are you doing this data normalization somewhere else? I think `development` is not always set and therefore `NODE_ENV` might be fasly...

```suggestion
          properties: { ...properties, projectType: strapi.projectType, environment: process.env.NODE_ENV || 'development' },
```

Reply:
That shouldn't be necessary, you can check there, [that's where this env is coming from](https://github.com/strapi/strapi/blob/b4cd5bf9a8489c3ec79b0a5f6ccd32c901519b2a/packages/core/admin/env.js#L26), the fallback has already been added.

Reply:
Hey @gu-stav, thanks for the question. @fdel-car kinda beat me to it and hopefully answered it :)